index.html

```diff
@@ -5850,11 +5864,16 @@ <h2 id="instantiation-of-AudioWorkletNode-and-AudioWorkletProcessor">
                   false).
                 </li>
                 <li>Perform <a href=
-                "https://tc39.github.io/ecma262/#sec-createdataproperty">CreateDataProperty</a>(O=<var>this</var>,
-                P=<var>paramName</var>, V=<var>audioParam</var>).
+                "https://tc39.github.io/ecma262/#https://tc39.github.io/ecma262/#sec-set-o-p-v-throw">
+                  Set</a>(O=<var>parameterMap</var>, P=<var>paramName</var>,
+                  V=<var>audioParam</var>).
                 </li>
               </ol>
             </li>
+            <li>Perform <a href=
+            "https://tc39.github.io/ecma262/#sec-createmethodproperty">CreateDataProperty</a>(O=<var>this</var>,
+            P="parameters", V=<var>parameterMap</var>).
+            </li>
```

Comment:
Along the lines of some earlier comments by Elliott, an AWN's `parameters` attribute should be a read-only slot that gets the parameter map, rather than a read/write data property. We don't want someone instantiating an AWN and then setting `node.parameters` to some other non-maplike object.